plugins/hydra_nevergrad_sweeper/README.md

```diff
@@ -0,0 +1,57 @@
+# Hydra Nevergrad Sweeper plugin
+
+#### Example of training using Nevergrad hyperparameter search
+```
+Using the following configuration:
+```yaml
+defaults:
+  - hydra/sweeper: nevergrad-sweeper
+hydra:
+  sweeper:
+    params:
+      budget: 100
+      num_workers: 10
+db: mnist
+lr: 0.01
+dropout: 0.6
+batch_size: 4
+```
+
+And running with:
+```text
+$ python example/dummy_training.py -m db=mnist,cifar batch_size=4,8,16 lr='Log(a_min=0.001,a_max=1.0)' dropout=0.0:1.0
+```
+
+Example of the outputs for the initialization and the first 5 evaluations (out of 100):
+
+```text
+[2020-02-21 10:03:21,496][HYDRA] NevergradSweeper(optimizer=OnePlusOne, budget=100, num_workers=10) minimization
```

Comment:
This is a bit noisy, can you edit to remove the timestamps?

Reply:
Not sure I understand, what should I update? The time stamp comes from the standard login in hydra right?

Reply:
Yes, just edit the output to make it less verbose:
remove prefixes like "[2020-02-21 10:03:21,496]"

You can also do it by re-running while overridng the formatting of the hydra logger from the command line but it's easier to just edit it out.